core/src/main/scala/tech/beshu/ror/accesscontrol/request/RequestContext.scala

```diff
@@ -75,11 +75,11 @@ object RequestContext extends Logging {
            history: Vector[Block.History])
           (implicit headerShow: Show[Header]): Show[RequestContext] =
     Show.show { r =>
-      def stringifyLoggedUser = {
+      def stringifyUser = {
         loggedUser match {
           case Some(DirectlyLoggedUser(user)) => s"${user.show}"
           case Some(ImpersonatedUser(user, impersonatedBy)) => s"${impersonatedBy.show} (as ${user.show})"
-          case None => "[user not logged]"
+          case None => r.attemptedUserName.map(name => s"attempted as '${name.value}'").getOrElse("[no info about user]")
```

Comment:
I don't think this is a good idea. We don't want to place eg. basic auth header value in logs. Audit is a different thing

Reply:
Yeah I was thinking the same, in logs let's log the attempted user, but not the whole auth header in case base http auth parsing fails.